docs/guide/deployment/coap-gateway.md

```diff
@@ -87,12 +87,6 @@ clients:
         caPool: "/data/certs/root_ca.crt"
         keyFile: "/data/certs/http.key"
         certFile: "/data/certs/http.crt"
-    oauth:
```

Comment:
Please document warning to this section: https://plgd.dev/architecture/component-overview/#cloud-registration that "Only JWT tokens are supported on the device."

Reply:
https://github.com/plgd-dev/plgd.dev/pull/15